dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/ActionPlan/Users.cshtml

```diff
@@ -45,10 +42,14 @@
                 <td>
                     @Html.ActionLink("Goals", "Index", "Goals", routeValues, new Dictionary<string, object>())
                 </td>
-                <td>
-                    @Html.ActionLink("Health data", "Index", "HealthData", routeValues, new Dictionary<string, object>())
-                </td>
-                <td>
+                 <td>
+                     @Html.ActionLink("Health data", "Index", "HealthData", routeValues, new Dictionary<string, object>())
+                 </td>
+
```

Comment:
it was there on purpose to visually group the adding of the startDate route value with the link below it